heartbeat/_meta/beat.yml

```diff
@@ -1,191 +1,20 @@
 ################### Heartbeat Configuration Example #########################
 
+# This file is an example configuration file highlighting only some common options.
+# The heartbeat.full.yml file in the same directory contains all the supported options
+# with detailed comments. You can use it for reference.
+#
+# You can find the full configuration reference here:
+# https://www.elastic.co/guide/en/beats/heartbeat/index.html
+
 ############################# Heartbeat ######################################
 
 # Configure monitors
 heartbeat.monitors:
-- type: icmp # monitor type `icmp` (requires root) uses ICMP Echo Request to ping
-             # configured hosts
-
-  # Monitor name used for job name and document type.
-  #name: icmp
-
-  # Enable/Disable monitor
-  #enabled: true
-
-  # Configure task schedule using cron-like syntax
-  schedule: '*/5 * * * * * *' # exactly every 5 seconds like 10:00:00, 10:00:05, ...
+- type: icmp
```

Comment:
If we switch to http as default, the discussion comes up which port should be monitored by default. In the default setup we expect 9200 to be available (ES). Current default is 80.